### DISCUSSION TITLE
Query ES timeout

### DISCUSSION THREAD

**Discussion by wangyjx** [apache/doris#46093] [A - General / Q&A]
# query any ES table leads to timeout
 Create ES catalog and show tables succeeded, but cannot query, cannot understand. 

## topology
They both runs inside k8s, Doris use host machine IP and port of ES container (Not K8S internal IP address) to create ES catalog.

## the error message:
RuntimeLogger W20241227 5:52:38.675930
3500 fragment mgr.cpp:549]report
err0
r<char>
日并
doris::HttpClient::execute(std::function<bool(void const*, unsigne
工#
doris::HttpClient::execute_post_request(std::__cxx11::basic_string<
at /var/local/ldb-toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../
>*】
2#
3#
4#
doris::ESScanReader::open()at /home/zcp/repo_center/doris_release/
doris::vectorized::NewEsScanner::open(doris::RuntimeState*)at /hom
doris::vectorized::ScannerScheduler::_scanner_scan(std::shared_ptr<
status :
errmsg is:[HTTP ERROR]Connection time
Failed to connect to ES server.
long)>const&)at /home/zcp/repo_center/doris_release/doris/be/src/common/stat
std::char_traits<char>,std::allocator<char>>const&,std::_cxx11::basic
har
/../../include/c++/11/bits/std_function.h:244
oris/be/src/common/status.h:491
/zcp/repo_center/doris_release/doris/be/src/common/status.h:491
oris::vectorized::ScannerContext>,std::shared_ptr<doris::vectorized::ScanTask>
out after 5001 milliseconds
is .h:@
string<char.
std::char traits<char>.
std::allocato
at /home/zcp/repocenter/doris_release/doris/be/s
rc/common/status.h:378
5# std::_Function_handler<void(),doris::vectorized::ScannerScheduler
:submit(std::shared ptr<doris::vectorized::ScannerContext>
std::shared ptr<dor
s::vectorized::ScanTask>)::$1::operator()()const
::{lambda()#1}>::_M_invoke(std:: Any_data const&)at /var/local/ldb-toolchain/b
n/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/shared_ptr_bas
.h:701
6# doris::ThreadPool::dispatch_thread()at /home/zcp/repo_center/doris
release/doris/be/src/util/threadpool.cpp:0
7#
doris::Thread::supervise_thread(void*)at /var/local/ldb-toolchain/
in/../usr/include/pthread.h:562
8栋
9#
-_clone
to coordinator:TNetworkAddress(hostname=172.20.80.2
port=9020)
juery id: af51715d9f0b43f9-8f338e3ea49fdb34,instance id: 0-0
response:
RuntimeLogger I20241227 5:52:38.676049
5631 fragment_mgr.cpp:730]
ry af51715d9f0b43f9-8f338e3ea49fdb34 instance af51715d9f0b43f9-8f338e3ea49fdb36
all done? true
Removing qu
RuntimeLogger I20241227 5:52:38.676064
5631 fragment_mgr.cpp:736]
Query af517
5d9f0b43f9-8f338e3ea49fdb34 finished

please give me some guidance.

**Comment by ixzc** [comment]
the error messages is 
`errmsg is:[HTTP ERROR]Connection time
Failed to connect to ES server.`
please confirm the connection between two databases
